langchain4j-core/src/main/java/dev/langchain4j/store/embedding/EmbeddingStore.java

```diff
@@ -55,6 +55,26 @@ public interface EmbeddingStore<Embedded> {
      */
     List<String> addAll(List<Embedding> embeddings, List<Embedded> embedded);
 
+    /**
+     * Removes an embedding from the store.
+     *
+     * @param id The unique identifier of the embedding to be removed.
+     * @return True if the embedding was successfully removed, false otherwise.
+     */
+    default boolean remove(String id) {
+        throw new UnsupportedOperationException("Not supported yet.");
+    }
+
+    /**
+     * Removes multiple embeddings from the store.
+     *
+     * @param ids A list of unique identifiers of the embeddings to be removed.
+     * @return A list of unique identifiers of the embeddings that were successfully removed.
+     */
+    default List<String> removeAll(List<String> ids) {
+        throw new UnsupportedOperationException("Not supported yet.");
+    }
+
     /**
      * Searches for the most similar (closest in the embedding space) {@link Embedding}s.
```

Comment:
What about adding `removeAll()` that will remove all embeddings?
I guess this might be useful in many cases:
- original datasource was updated and one wants to simply re-ingest everything instead of figuring out what was changed and what should be removed and re-ingested again
- one might want to change the embedding model and needs to re-ingest everything

Another useful method would be `removeAll(Filter)` which is supported by some embedding stores (e.g. [Milvus](https://milvus.io/docs/insert-update-delete.md#Delete-entities), [Pinecone](https://docs.pinecone.io/guides/data/delete-data#delete-specific-records-by-metadata)), but this is probably for another PR.

WDYT?